noxfile.py

```diff
@@ -29,6 +29,9 @@
 DEFAULT_PYTHON_VERSION = "3.8"
 SYSTEM_TEST_PYTHON_VERSIONS = ["2.7", "3.8"]
 UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.6", "3.7", "3.8", "3.9"]
+CONFORMANCE_TEST_PYTHON_VERSIONS = ["2.7", "3.8"]
+
+_DEFAULT_STORAGE_HOST = "https://storage.googleapis.com"
```

Comment:
Changes to `noxfile.py` need to be preserved by adding appropriate `s.replace` entries to `owlbot.py`:  otherwise, they will vanish in the next bot-generated PR.

Reply:
Also, I reckon we can avoid running the new conformance tests on Python 2, unless there is a compelling reason to believe they would fail differently.

Reply:
Thanks @tseaver for the comments. I agree we can remove Python 2 from the conformance tests. Will look into the owlbot user guide and add the entries.

Reply:
@tseaver Will the `noxfile.py` changes vanish with the existing [exclusion](https://github.com/googleapis/python-storage/blob/6486010c5835d25a9284c6b17730a3c31177e55a/owlbot.py#L40-L42)? 
```
s.move(
    templated_files, excludes=["docs/multiprocessing.rst", "noxfile.py", "CONTRIBUTING.rst"],
)
```
Is there any Owlbot/Synthtool documentation towards how to add `s.replace` entries with noxfile changes? So far I've only come to a [pydoc](https://htmlpreview.github.io/?https://github.com/googleapis/synthtool/blob/master/synthtool/pydoc.html). Would it be something like so?
```
s.replace(
    "google/cloud/python-storage/noxfile.py",
    "UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.6", "3.7", "3.8", "3.9"]",
    "UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.6", "3.7", "3.8", "3.9"]\n"
    "CONFORMANCE_TEST_PYTHON_VERSIONS = ["3.8"]"
)
```

Reply:
A couple of scars I have from using `s.replace`:

- The second argument is a regex, so you need to escape a bunch of special characters with `\`, which means the string should be "raw" (see https://docs.python.org/3/library/re.html).
- I generally prefer to use triple-quoted strings for any multi-line replacements:  it is easier to read.

So, for your example, it would be:

```python
s.replace(
    "google/cloud/python-storage/noxfile.py",
    r"UNIT_TEST_PYTHON_VERSIONS = \["2\.7", "3\.6", "3\.7", "3\.8", "3\.9"\]",
    """\
UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.6", "3.7", "3.8", "3.9"]
CONFORMANCE_TEST_PYTHON_VERSIONS = ["3.8"]"""
)
```

Reply:
Got it, @tseaver. I was following the pydoc and thought I could pass in `before` as a `str` [```replace(sources: Iterable[Union[str, pathlib.Path]], before: str, after: str, flags: int = <RegexFlag.MULTILINE: 8>) -> int```](https://htmlpreview.github.io/?https://github.com/googleapis/synthtool/blob/master/synthtool/pydoc.html)

I'm still kind of confused on the s.replace entries. I've added [a huge chunk of session conformance test in `noxfile.py`](https://github.com/googleapis/python-storage/blob/53b9f7d20dc5feefbcddbd8c8f52e6507acd6ddf/noxfile.py#L154-L185). How would I preserve that change in owlbot.py?